vignettes/introduction.Rmd

```diff
@@ -41,7 +41,7 @@ This will do a number of things:
 
 1. Changes your working directory to `${CODE}/project_name`
 
-2. Runs a project-specific R init script, if you have one saved at `${CODE}/project_name/00-init.R`.
+2. Runs a project-specific R init script, if you have one saved at `${CODE}/project_name/src/00-init.R`.
```

Comment:
Is this changing, or were the docs wrong before? If it's a logical change, we should add this to `NEWS`

Reply:
The docs were wrong - the vignette originally failed. `src` is the default `scriptSubdir`.

Reply:
OK cool